server.go

```diff
@@ -705,6 +704,25 @@ func hasServices(advertised, desired wire.ServiceFlag) bool {
 	return advertised&desired == desired
 }
 
+// isSupportedNetAddressTypeV1 returns whether the provided address manager
+// network address type is supported by the addr wire message.
+func isSupportedNetAddressTypeV1(netAddressType addrmgr.NetAddressType) bool {
```

Comment:
Similar to comments on the previous commit, I would suggest a consistent param name `addrType`.